tests/SprykerEcoTest/Zed/Braintree/Business/BraintreeFacadeCheckoutPostSaveHookTest.php

```diff
@@ -0,0 +1,145 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEcoTest\Zed\Braintree\Business;
+
+use Braintree\Transaction\CreditCardDetails;
+use Generated\Shared\Transfer\CheckoutResponseTransfer;
+use SprykerEco\Zed\Braintree\BraintreeConfig;
+use SprykerEco\Zed\Braintree\Business\Order\Saver;
+use SprykerEco\Zed\Braintree\Business\Payment\Transaction\PaymentTransaction;
+use SprykerEco\Zed\Braintree\Dependency\Facade\BraintreeToMoneyFacadeBridge;
+use SprykerEco\Zed\Braintree\Dependency\Facade\BraintreeToMoneyFacadeInterface;
+use SprykerEco\Zed\Braintree\Persistence\BraintreeEntityManager;
+
+/**
+ * Auto-generated group annotations
+ *
+ * @group SprykerEcoTest
+ * @group Zed
+ * @group Braintree
+ * @group Business
+ * @group Facade
+ * @group BraintreeFacadeCheckoutPostSaveHookTest
+ * Add your own group annotations below this line
+ */
+class BraintreeFacadeCheckoutPostSaveHookTest extends AbstractFacadeTest
+{
+    /**
+     * @return void
+     */
+    public function testCheckoutPostSaveHookWithSuccessfulResponse(): void
```

Comment:
usually we need to cover facade with at lesat to test cases, positive and negative
also, if any transfer properties required, we need to add a test with `->expecrtException()`
here `quote.payment` seems to be required

Reply:
Additional test was added.